images/miq-app/docker-assets/container-scripts/container-deploy-common.sh

```diff
@@ -301,7 +207,7 @@ function backup_pv_data() {
   (
     echo "== Initializing PV data backup =="
 
-    rsync -av --exclude 'log' "${PV_CONTAINER_DATA_DIR}" "${PV_CONTAINER_DATA_REGION_DIR}" "${PV_BACKUP_DIR}/backup_${PV_BACKUP_TIMESTAMP}"
+    rsync -av --exclude 'log' "${PV_CONTAINER_DATA_DIR}" "${PV_BACKUP_DIR}/backup_${PV_BACKUP_TIMESTAMP}"
```

Comment:
@carbonin Looks good, sending everything except logs from server pv data container data dir to backup dir on the same PV. Before we were processing both PVs on the same rsync one-liner.